index.html

```diff
@@ -10,25 +10,148 @@
+        <ul class="title">
+            <li class="lang1">HTML</li>
+            <li class="lang2">CSS</li>
+            <li class="lang3">JavaScript</li>
+        </ul>
+        <div class="see">
+        <a href="#" class="text">See Project</a>
+        </div>
+    </div>
+
+    <!-- About me Section -->
+    <div class="about">
+        <div class="base">
+            <div class="top">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social">
+            <a href="#">LET'S CONNECT</a>
+            <ul class="icons">
                 <li><img src="/image/Vector.svg"></li>
                 <li><img src="/image/Linkedin icon.svg"></li>
                 <li><img src="/image/Vector (1).svg"></li>
                 <li><img src="/image/Vector (2).svg"></li>
                 <li><img src="/image/Vector (3).svg"></li>
-
             </ul>
-    </section>
-</body>
+        </div>
+        <div class="btn">
+            <a href="#" class="resume">Get my resume</a>
+        </div>
+        <div class="skill_list">
+            <div class="title1">
+                <h1>Languages</h1>
+                <a href="#" class= "fa fa-angle-down"></a>
+                <div class="langlist">
+                <ul class="list1">
+                    <img src="/image/Ellipse 1.png">
+                    <p>JavaScript</p>
+                </ul>
+                <ul class="list2">
+                    <img src="/image/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </ul>
+                <ul class="list3">
+                    <img src="/image/Ellipse 1 (2).png">
+                    <p>CSS</p>
+                </ul>
+                </div>
+            </div>
+            <div class="title2">
+                <h1>Frameworks</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+            <div class="title3">
+                <h1>Skills</h1>
+                <a href="#" class= "fa fa-angle-right"></a>
+            </div>
+        </div>
+        </div>
+    </div>
+
+    <!-- Contact me -->
+    <div class="contact">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+            I’d love to help with it! </p>
+        <input type="text" id="fname" name="fname" value="Lorem Ipsum">
+        <input type="text" id="email" name="email" value="Loremipsum@gmail.com">
+        <input type="text" id="msg" name="msg" value="Write your message here...">
+        <div class="submit">
+            <a href="#" class="tt">Get in touch</a>
+        </div>
+    </div>
```

Comment:
- [ ] [OPTIONAL] This section can be removed since it is not part of the requirement of this milestone. It will also allow you to focus on important features of the user stories :hugs: